rewritingTask3/functions.c

```diff
@@ -0,0 +1,39 @@
+#include "functions.h"
+#include <stdio.h>
+#include <stdbool.h>
+
+#define MAX_NUMBERS 100
+
+int readFromFile(const char* filename, int* array) {
```

Comment:
```suggestion
int readFromFile(const char* fileName, int* array) {
```
camelCase требует разделять слова заглавными буквами